scripts/npm-publish.sh

```diff
@@ -5,8 +5,9 @@ set -e
 set -o pipefail
 
 if [[ -z $NPM_TOKEN ]]; then
-  echo "Notice: NPM_TOKEN environment variable not set. Skipping npm publishing."
-  exit 1
+  echo "Notice: NPM_TOKEN environment variable not set. Performing --dry-run publish."
```

Comment:
Here it seems like you _do_ want to be more technical, so perhaps "Running \`npm publish --dry-run\`."?